### PR TITLE
Update faucet url for the ghostnet faucet

### PR DESCRIPTION
**Pull request by serjonya-trili**

### CODE REVIEW

**Review by asiia-trilitech — COMMENTED** [review]
Backups might contain the old url, should we somehow handle this as well?

File: src/utils/redux/migrations.ts [hunk 1]
```diff
@@ -16,6 +16,18 @@ export const mainStoreMigrations = {
       draft.announcement = announcementsInitialState;
     }),
   2: identity,
+  3: (state: any) =>
+    produce(state, (draft: any) => {
+      if (draft.networks.current.name === "ghostnet") {
+        draft.networks.current.buyTezUrl = "https://faucet.ghostnet.teztnets.com/";
+      }
+      for (const network of draft.networks.available) {
+        if (network.name === "ghostnet") {
```
[the remark this answers is not in the document]
Comment (serjonya-trili) [reply src/utils/redux/migrations.ts:25]: another network might use the same URLs, but it'd be a custom network that we don't care about. this one is hardcoded by us and must be correct

**Comment by serjonya-trili** [comment]
> Backups might contain the old url, should we somehow handle this as well?

when we import a backup it has an old version and will also go through the migrations

**Review by asiia-trilitech — APPROVED** [review]
Thanks for the clarifications, Sergey!

LGTM 👍

**Pull request merged**